pints/toy/_distributions.py

```diff
@@ -0,0 +1,186 @@
+#
+# Various toy pdfs to use to test samplers.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+from scipy.stats import multivariate_normal
+
+
+class TwistedGaussianLogPDF(pints.LogPDF):
```

Comment:
Needs:

- [ ] An example in a notebook
- [ ] A test
- [ ] Sphinx documentation